File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let occurrenceMap = {}
  for(let i=0; i<nums.length; i++) {
      if(nums[i] in occurrenceMap)
          return true
        occurrenceMap[nums[i]] = 1
  }  
  return false
};

File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Map<Integer, Integer> occurrenceMap = new HashMap<>();
        
        for(Integer n: nums) {
            if(occurrenceMap.containsKey(n)) {
                return true;
            } else 
                occurrenceMap.put(n ,1);
        }
        
        return false;
    }
}